code/datums/ruins/space.dm

```diff
@@ -182,3 +182,71 @@
 	name = "Crashed Wizard Shuttle"
 	description = "A shuttle of the Wizard Federation, sent out to crush some wandless scum. Unfortunately, the pilot suffered a magic-related accident and the shuttle crashed into a nearby asteroid."
 	cost = 2
+
+/datum/map_template/ruin/space/abandonedtele
+	id = "abandonedtele"
+	suffix = "abandonedtele.dmm"
+	name = "Abandoned Teleporter"
+	description = "An old teleporter, seemingly part of what used to be a larger satellite."
```

Comment:
Do these descriptions appear anywhere ingame? If so, where?

Reply:
On the admin `Jump To Ruin` verb, but nowhere else. All the others had descriptions and names so I felt it was appropriate.